user/homework.go

```diff
@@ -57,6 +59,36 @@ func (hw *Homework) Save() error {
 	return nil
 }
 
+// SaveAfterFetch filters out problems for tracks the user has yet to start
+// before saving them all to disk.
+func (hw *Homework) SaveAfterFetch(numArgs int) error {
+	hw.RejectUnstartedTracks(numArgs)
+	if len(hw.Items) == 0 {
+		fmt.Println(`
+You have yet to start a language track!
+View all available language tracks with "exercism tracks"
+Fetch exercises for your first track with "exercism fetch LANGUAGE"`)
+	}
+	return hw.Save()
+}
+
+// RejectUnstartedTracks removes any items that are part of tracks the user
+// has yet to start and doesn't have a folder for on their local machine. This
+// only happens when a user calls `exercism fetch` without any arguments.
+func (hw *Homework) RejectUnstartedTracks(numArgs int) {
```

Comment:
OK, now I'm nitpicking _really_ hard! They're not necessarily unstarted in this scenario, they're just... I don't know... inactive? If I started a track but then deleted the directory off my machine, it would not write new exercises to it.

I'm also thinking the `numArgs` is going to confuse me down the line when I've forgotten this discussion.

I don't know the right solution, so I'm just going to think out loud.

The arguments are arguments to the command when the user calls `exercism fetch`. We're using the argument count as a flag: either do something or don't do it (right? Or am I forgetting something?).

If it's a flag, then it's a flag for... filtering?

A boolean makes a good flag, but I'm not sure what to name it. `isSomething`? But I'm not sure what makes a good _something_ in this context.

Any suggestions?